test/Driver/static-executable.swift

```diff
@@ -0,0 +1,10 @@
+// Create a self contained binary
+// REQUIRES: OS=linux-gnu
+// REQUIRES: static_stdlib
+print("hello world!")
+// RUN: %empty-directory(%t)
+// RUN: %target-swiftc_driver -static-executable -o %t/static-executable %s
+// RUN: %t/static-executable | %FileCheck %s
+// RUN: not ldd %t/static-executable | %FileCheck %s --check-prefix=LDD
```

Comment:
Please do not use `ldd` - that is a security issue.  Please use `llvm-readelf` instead to check the binary type.

Reply:
Ah, thanks!